libass/ass_render.h

```diff
@@ -191,6 +191,9 @@ typedef struct {
     CombinedBitmapInfo *combined_bitmaps;
     unsigned n_bitmaps;
     double height;
+    int border_top;
+    int border_bot;
```

Comment:
I'd use "bottom" here; "bot" isn't exactly a common abbreviation for this.

Reply:
Done.